plugins/auth_base/fps_auth_base/__init__.py

```diff
@@ -9,6 +9,7 @@
     current_user = auth["current_user"]
     update_user = auth["update_user"]
     websocket_auth = auth["websocket_auth"]
+
```

Comment:
Not needed.

Reply:
Sometimes having spaces helps when reading code

Reply:
Then it should be coherent with the rest of the code base. But IMO there is no space needed before an `except`.